### PR TITLE
Setup preact and jest/enzyme

### PR DESCRIPTION
**Pull request by mnunemacher-dm**

### CODE REVIEW

File: src/components/JustNotSorry.js [hunk 1]
```diff
@@ -128,12 +129,14 @@ class JustNotSorry extends Component {
 
       this.setState((state) => {
         let warnings = state.warnings;
```
Comment (sbrudz) [inline src/components/JustNotSorry.js:131]: `let warnings = state.warnings;` isn't needed anymore.  You can add a `const` to the next line instead.

File: src/components/JustNotSorry.js [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+        rangeToHighlight: range,
+      };
+
+      this.setState((state) => {
+        let warnings = state.warnings;
+        warnings = state.warnings.concat(newWarning).filter(function (warning) {
+          if (warning.rangeToHighlight.startContainer) {
+            return (
+              warning.rangeToHighlight.startContainer.textContent !=
+              newWarning.message
```
Comment (sbrudz) [inline src/components/JustNotSorry.js:136]: This should be a `!==`

File: src/components/JustNotSorry.js [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+  addWarnings(node) {
+    WARNING_MESSAGES.map((warning) => {
+      this.addWarning(node, warning.keyword, warning.message);
+    });
+  }
+
+  render() {
+    const warningList = this.state.warnings.map((warning) =>
+      ReactDOM.createPortal(
+        <Warning class=".jns-warning" key={warning.keyword} value={warning} />,
```
Comment (sbrudz) [inline src/components/JustNotSorry.js:156]: `class` isn't getting picked up here.  It should be `className`.  That doesn't seem to be impacting the UI, though, so maybe it's not needed at all.

**Comment by yutingcxiang** [comment]
Changes have been made, thanks for reviewing @sbrudz !

**Pull request merged**